android/app/proguard-rules-debug.pro

```diff
@@ -0,0 +1,93 @@
+# Add project specific ProGuard rules here.
```

Comment:
There's content in this file which looks to me to be taken from somewhere. I imagine the origin is proguard-rules.pro. Seeing that `proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'` lists a couple of files, I imagine we can have a common ProGuard rule file for debug and release builds. Alternatively, I think I read ProGuard has an `-include` so we could include the common file in the debug and release-specific files.

Reply:
Done!